### ISSUE TITLE
Add a Raw HTML layout element

### ISSUE THREAD

**Issue by sebastienros** [OrchardCMS/Orchard#5773]
There is currently no way to add custom HTML in a layout without having it altered by TinyMce.
The editor could be a plain text area or CodeMirror.

**Comment by Piedone** [comment]
Instead of a new element we could have a tab for example with a raw code editor?

**Comment by sfmskywalker** [comment]
And then synchronize with TinyMce's version of the HTML? Can we prevent it from messing with the HTML when you switch back to the TinyMce tab?

**Comment by Piedone** [comment]
Yes, sync it. Although I don't know I guess somehow it's possible to prevent TinyMCE to do any changes (the main issue here I think is that it requires a container tag around plain text; but if you already have a container like a div it won't put another one, a p by default, around it).

**Comment by sfmskywalker** [comment]
Or, as suggested by @bleroy, how about we get rid of the various html/text/paragraph elements in favor of a single `Text` element that has various flavors and properties?

If you think that's interesting, s few implementation questions pop up in my mind: when you drag this new `Text` element onto the canvas and the editor appears, would you specify the flavor right there and then? And when you change the flavor, would the dialog's contents be refreshed to render the appropriate editor for the chosen flavor?

Whatever the implementation there, what about taking the same approach as done with the various Row toolbox elements, which are pre-configured to add a number of child columns? So even though we would have a single `Text` element, its toolbox representation could show a bunch of pre-configured items: Text, Html and Markdown. The Text element would have options such as "HTML Encode (y/n)".
